public/components/Flyout/flyouts/components/AlertsDashboardFlyoutComponent.js

```diff
@@ -570,6 +547,10 @@ export default class AlertsDashboardFlyoutComponent extends Component {
         displayTableTabs = false;
         break;
     }
+    const monitorUrl = `${PLUGIN_NAME}#/monitors/${monitor_id}${
```

Comment:
Do the links to the monitors work as expected for non-composite monitors? Just curious because this line in the snapshot suggests the url may not be formed as intended
[https://github.com/opensearch-project/alerting-dashboards-plugin/pull/664/files#diff-b2eb7f8fa8b51f715cfb266824b[…]c8b09802d1bb87fabe908e5b3b3a29R74](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/664/files#diff-b2eb7f8fa8b51f715cfb266824bff107e2c8b09802d1bb87fabe908e5b3b3a29R74)

Reply:
Fixed this